src/Storages/ObjectStorageQueue/ObjectStorageQueueIFileMetadata.cpp

```diff
@@ -55,6 +55,7 @@ void ObjectStorageQueueIFileMetadata::FileStatus::onProcessing()
     processing_start_time = now();
     processing_end_time = {};
     processed_rows = 0;
+    last_exception = {};
```

Comment:
Added clearing of the exception, because otherwise it messes up `~ObjectStorageQueueIFileMetadata` in a way that we do not call `onFailed` as we see `exception_message.empty() == false`, but `state` can be actually `Processing` (in case of exception before we finish processing the file), so it was possible to get stuck in processing state.